collections/_posts/2024-02-20-Multi-Modal_VAE_for_normative_modelling.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: review
+title: "Multi-modal Variational Autoencoders for normative modelling across multiple imaging modalities"
+tags: autoencoders, multimodal, Product of Experts
+author: "Romain Deleat-besson"
+cite:
+    authors: "Ana Lawry Aguila, James Chapman, Andre Altmann"
+    title:   "Multi-modal Variational Autoencoders for normative modelling across multiple imaging modalities"
+    venue:   "MICCAI 2023"
+pdf: "https://arxiv.org/pdf/2303.12706.pdf"
+---
+
+
+# Notes
+
+Link to Tutorial [VAE](https://creatis-myriad.github.io/tutorials/2022-09-12-tutorial-vae.html)
+
+No Github repository.
+
+Figure to help understand Multi-modal VAE (First part is the Product of Experts loss. Second part is the Mixture of Experts loss.):
+
+![](/collections/images/Multi-modal_VAE_for_normative_modelling/fig_explicative.jpg)
+*Figure adapted from A. Salvador*
+
+&nbsp;
+
+
+# Highlights
+
+In this paper, the authors have two contributions:
```

Comment:
I don't like the formulation "have contributions". Maybe something more like propose/introduce/etc.
```suggestion
In this paper, the authors introduce two contributions:
```